src/nwm_routing/src/nwm_routing/preprocess.py

```diff
@@ -252,12 +252,16 @@ def nwm_initial_warmstate_preprocess(
     if break_network_at_waterbodies:
         ## STEP 3c: Handle Waterbody Initial States
         # TODO: move step 3c into function in nnu, like other functions wrapped in main()
-      
         start_time = time.time()
-    
         LOG.info("setting waterbody initial states ...")
 
-        if restart_parameters.get("wrf_hydro_waterbody_restart_file", None):
+        if restart_parameters.get("lite_waterbody_restart_file", None):
+            
+            waterbodies_initial_states_df, _ = nhd_io.read_lite_restart(
+                restart_parameters['lite_waterbody_restart_file']
+            )
```

Comment:
If a lite waterbody restart file is provided, it is read-in with preference over `HYDRO_RST` or `.csv` files